tasks/build/local/build.js

```diff
@@ -23,18 +23,25 @@ cli(optionsSpecification, async options => {
         ? `windows_amd64`
         : `linux_amd64`
   try {
-    const secret = await init(options, environment)
+    // remove existing config
+    if (
+      options.overwrite &&
+      fs.existsSync(appDir + `/builds/testnets/local-testnet`)
+    ) {
+      let out = await makeExec(`rm -r builds/testnets/local-testnet`)
+      out && console.log(out)
```

Comment:
I appreciate the brevity here but `&&` is traditionally reserved for values that don't produce side effects.  It's considered better form to write this, even though it's longer:

```
if (out) {
  console.log(out)
}
```

I'm sorry I don't have a citation for this, it's just from the years of accumulated wisdom in my brain.